cmd/kops/create_cluster.go

```diff
@@ -843,7 +845,11 @@ func RunCreateCluster(f *util.Factory, out io.Writer, c *CreateClusterOptions) e
 	}
 
 	if len(c.AdminAccess) != 0 {
-		cluster.Spec.SSHAccess = c.AdminAccess
+		if len(c.SSHAccess) != 0 {
+			cluster.Spec.SSHAccess = c.SSHAccess
+		} else {
+			cluster.Spec.SSHAccess = c.AdminAccess
+		}
```

Comment:
This is good - thank you for keeping the existing behaviour.  But ... are we missing the case where we set ssh-access but not admin-access ?

You could also add a create_cluster integration test; you create a new directory here https://github.com/kubernetes/kops/tree/master/tests/integration/create_cluster (copy one of the existing ones) and then reference it here https://github.com/kubernetes/kops/blob/master/cmd/kops/create_cluster_integration_test.go.  But I'm also happy to write that test (and assign you the code review :-) ), as I'd love to get `--ssh-access` into 1.7.0).

Blocker to merge: the case where len(adminaccess) == 0 but len(sshAccess) != 0  (I think!)